eth/downloader/downloader.go

```diff
@@ -18,61 +18,64 @@
 package downloader
 
 import (
-	"bytes"
 	"errors"
 	"math"
 	"math/big"
-	"math/rand"
 	"sync"
 	"sync/atomic"
 	"time"
 
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/core"
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/event"
 	"github.com/ethereum/go-ethereum/logger"
 	"github.com/ethereum/go-ethereum/logger/glog"
-	"gopkg.in/fatih/set.v0"
 )
 
 const (
-	eth60 = 60 // Constant to check for old protocol support
-	eth61 = 61 // Constant to check for new protocol support
+	eth61 = 61 // Constant to check for old protocol support
```

Comment:
(1) sounds reasonnable!

(2) i meant that `protocolVersion` only defines codes for `PoC-1` to `PoC-6`, I guess my question is how can I specify `PV62` (0x??) for my `Status` message? Or is `PoC-6` <=> `PV62`? Hopefully that makes sense...